.standard-GEM.md

```diff
@@ -42,13 +42,13 @@ The URL can be the link to the publication/pre-print/website where the model is
 
 Repository workflow
 -------------------
-- [ ] 🟥 Git branches  
-The GEM repository must have at least two branches: _master_ and _develop_.
+- [x] 🟥 Git branches  
+The GEM repository must have at least two branches: _master_ and _devel_.
```

Comment:
For consistency, the text in the standard should not be changed - otherwise we risk ending up with distinct standards. The [git workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) refers to the name _develop_ not _devel_.

Reply:
The pick of _devel_ is for consistency with many existing GEM-type repos, e.g. `Human-GEM`, `yeast-GEM`. And the branch name should be self-adaptive within a repo rather than strictly defined, because sometimes there will be cases for a [change](https://github.com/MetabolicAtlas/standard-GEM/issues/6).

Reply:
There are many git repositories that have a development branch, and that branch is generally called _develop_. The non-standard name _devel_ was most likely adopted by the GEM repositories in SysBioChalmers because of [old instructions](https://wiki.portal.chalmers.se/sysbio/ComputationalLab/SOPForUploadingGEMsOnSysBioChalmersGithub). Maintaining consistency is not required, because these repositories do not rely on each other's branching naming scheme. So, I would encourage Mouse-GEM to adopt a general convention, and nudge the other repositories to do the same.

Reply:
Sounds good, and agree to adopt the _develop_ at some point.

Reply:
Sounds good. New _develop_ branch is created and start to be in use immediately. The existing _devel_ will be removed later at some point.